src/hooks/use-on-key-down.js

```diff
@@ -0,0 +1,13 @@
+import { useRef, useEffect } from "react";
+
+export const useOnKeyDown = handler => {
+  const ref = useRef();
+  ref.current = handler;
```

Comment:
Whats the difference between use-is-key-pressed.js and this use-on-key-down.js, does the first file not cover both?

Reply:
useOnKeyDown only invokes a handler on an keydown event, useIsKeyPressed returns a Boolean value that can be used in the UI.